src/Tasks/ProcessJobQueueTask.php

```diff
@@ -43,6 +47,26 @@ public function run($request)
 
         $service = $this->getService();
 
+        // Ensure that log messages are visible when executing this task on CLI.
+        // TODO Replace with BuildTask logger: https://github.com/silverstripe/silverstripe-framework/issues/9183
+        if (Environment::isCli()) {
+            $logger = $service->getLogger();
+
+            // Assumes that general purpose logger usually doesn't already contain a stream handler.
+            $errorHandler = new StreamHandler('php://stderr', Logger::ERROR);
+            $standardHandler = new StreamHandler('php://stdout');
+
+            // Avoid double logging of errors
+            $standardFilterHandler = new FilterHandler(
+                $standardHandler,
+                Logger::DEBUG,
+                Logger::WARNING
+            );
+
+            $logger->pushHandler($standardFilterHandler);
```

Comment:
do we need to test if the $logger is not null here?

Reply:
Shouldn't be the case any more, see previous comment.